runtimes/pythonrt/worker_grpc_handler.go

```diff
@@ -373,6 +375,37 @@ func (s *workerGRPCHandler) Unsubscribe(ctx context.Context, req *pb.Unsubscribe
 	}
 }
 
+func (s *workerGRPCHandler) EncodeJWT(ctx context.Context, req *pb.EncodeJWTRequest) (*pb.EncodeJWTResponse, error) {
+	// GitHub's JWTs must be signed using the RS256 algorithm:
+	// https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-json-web-token-jwt-for-a-github-app
+	if req.Algorithm != jwt.SigningMethodRS256.Name {
```

Comment:
why do you even pass it in in the first place oh well

Reply:
in case they value is incorrect - we should get a clear error message.
Why check here and not before? To keep the passthrough code as generic as possible, in case this criterion changes in the future.